### DISCUSSION TITLE
JPA vs ???. 다른 것들과의 비교. 상황에 따라서 성능적으로 비교하기

### DISCUSSION THREAD

**Discussion by DohunHyun** [100-hours-a-week/discussions#19] [Deepdive]
# JPA에 대해서 간단하게, 성능적으로 분석해보기

## ??? 의 후보
1. Mybatis
2. JPQL
3. JDBC
4. QueryDSL
5. Spring data JPA
6. hibernate

### 2. 특정 상황을 정해서 그 상황에 따라서 성능 비교
특정 상황
1. 현재 프로젝트에서 사용하는 쿼리 작업에 대해서 비교
2. 복잡한 쿼리에 대해서 비교
    2-1. 현재 프로젝트에서 사용하는 쿼리 중에 가장 복잡한 쿼리.
3. 각 후보에게 유리한 경우를 가정한 후 비교
### 3. JPA가 이러한 성능을 나타내는 이유 알아보기. (코드 분석하고 특정 상황에 따라 어떤 이유 때문에 그런 결과가 나오는지)

**Comment by startupcodekr** [comment]
제목에 본인 프로젝트 시나리오에서 문제점으로 인한 내용이 필요합니다!
